hook.go

```diff
@@ -13,6 +13,7 @@ func (dummyFormatter) FormatQuery(b []byte, query string, params ...interface{})
 	return append(b, query...)
 }
 
+// QueryEvent ...
```

Comment:
Hard to say if you got bored or missed this :) Anyway I guess we can address this later.